### ISSUE TITLE
'Radius server name' can/should include certificate hashes

### ISSUE THREAD

**Issue by troyphil** [MicrosoftDocs/memdocs#3206]
When configuring Android Enterprise devices to connect to Enterprise Wi-Fi (EAP - TLS) and authenticating with device issued certificates, we discovered that 'Radius server name' list did not just need the NPS FQN but also the thumbprint of the certificate presented by NPS. In our case we added both SHA-1 and SHA-256 hashes from the certificate used by NPS.

Please update the documentation to specify that certificate hashes are valid for the 'Radius server name' list, and ideally the situations when needed and/or appropriate.

Cheers
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3ba02cac-ed62-ad81-b30f-0c72eae1ec06
* Version Independent ID: 336ccf7f-9ad9-05a4-6755-dcfa4ca71920
* Content: [Wi-Fi settings for Android Enterprise and kiosk devices - Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/configuration/wi-fi-settings-android-enterprise?source=recommendations)
* Content Source: [memdocs/intune/configuration/wi-fi-settings-android-enterprise.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/configuration/wi-fi-settings-android-enterprise.md)
* Service: **microsoft-intune**
* Sub-service: **configuration**
* GitHub Login: @MandiOhlinger
* Microsoft Alias: **mandia**

**Comment by yogkumgit** [comment]
@MandiOhlinger Please help us in resolving this issue. Thanks

**Comment by MandiOhlinger** [comment]
@troyphil Good point. We updated the doc to include some additional info. The changes should be live within the next couple of days. Thanks!

**Comment by sysadm216** [comment]
@troyphil What format did you use for the SHA1 and SHA256 hashes?
Did you add them each as separate rows and did you delimit the hashes with colons?
